app/connector/mongodb/src/test/java/io/syndesis/connector/mongo/MongoDBConnectorCappedCollectionConsumerAllOptionsTest.java

```diff
@@ -86,7 +82,7 @@ public void mongoTest() throws Exception {
                 JsonNode jsonNode = MAPPER.readTree(doc.get(0));
                 Assertions.assertThat(jsonNode).isNotNull();
             } catch (IOException ex) {
-                log.error("Test failed because: ",ex);
+                log.error("Test failed because: ", ex);
```

Comment:
Perhaps it would be better if the test fails and the error is seen in the unit test report, then we don't need to read the logs and correlate test to exception.